docs/source/main/building-security-tools-on-mythx.rst

```diff
@@ -302,6 +302,27 @@ The output contains a list of issues with title, short description and long desc
 - The `meta` field contains meta information about the analysis run.
 
 
+It is recommended that users submit both bytecode and source code to obtain a full analysis, but in the case that no source code is provided, ie: only the submitting the deployed bytecode. MythX will return the following result:
```

Comment:
I would change this to "i.e., by only submitting the (creation) bytecode. MythX will return a result such as the following:".

Reply:
Another suggestion:

We recommend that users submit both bytecode and source code to obtain a full analysis. If only the creation bytecode is given, and not the source code, MythX will return a result like the folllowing:

Reply:
@rocky Yeah, that's also fine with me.